docs/reference/api/platform/MemoryStats.md

```diff
@@ -1,11 +1,15 @@
 ## MemoryStats
 
-[Add description here.]
+Mbed OS provides a set of functions that you can use to capture the memory statistics at runtime. `mbed_stats.h` declares these functions. You can use memory statistics functions to capture heap usage, cumulative stack usage or stack usage per thread at runtime. To enable memory usage monitoring, you must build Mbed OS with the following macros.
 
-### MemoryStats class reference
+- MBED_MEM_TRACING_ENABLED
+- MBED_HEAP_STATS_ENABLED
+- MBED_STACK_STATS_ENABLED
 
-[Add embedded class reference here. If you've never done this, please talk to your editor.]
+### MemoryStats functions reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/v5.6/mbed-os-api-doxy/group__platform__stats.html)
 
 ### MemoryStats example
 
-[Add example here.]
+[![View Example](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-platform-utils)](https://github.com/ARMmbed/mbed-os-example-platform-utils) 
```

Comment:
@sg- Should this be in the Mbed example teams folder like the other API examples?